Sources/Kevin/Accounts/BankSelection/KevinBankSelectionView.swift

```diff
@@ -74,78 +67,27 @@ internal class KevinBankSelectionView : KevinView<KevinBankSelectionState> {
-        chevron.translatesAutoresizingMaskIntoConstraints = false
-        chevron.centerYAnchor.constraint(equalTo: countrySelectionContainer.centerYAnchor).isActive = true
-        chevron.rightAnchor.constraint(equalTo: countrySelectionContainer.rightAnchor, constant: -18).isActive = true
-        chevron.widthAnchor.constraint(equalToConstant: 14).isActive = true
-        chevron.heightAnchor.constraint(equalToConstant: 14).isActive = true
+        tableView.register(CountrySelectionCell.self, forCellReuseIdentifier: "CountrySelectionCell")
+        tableView.register(BankCell.self, forCellReuseIdentifier: "BankCell")
+        tableView.register(BankHeaderCell.self, forCellReuseIdentifier: "BankHeaderCell")
+        tableView.register(EmptyStateCell.self, forCellReuseIdentifier: "EmptyStateCell")
         
-        let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.onCountrySelectionClicked(_:)))
-        countrySelectionContainer.addGestureRecognizer(tapRecognizer)
-    }
-    
-    private func initBankSelection() {
-        bankSelectionLabel.text = "window_bank_selection_select_bank_label".localized(for: Kevin.shared.locale.identifier).uppercased()
-        bankSelectionLabel.font = Kevin.shared.theme.sectionStyle.titleLabelFont
-        bankSelectionLabel.textColor = Kevin.shared.theme.generalStyle.secondaryTextColor
-        addSubview(bankSelectionLabel)
-        bankSelectionLabel.translatesAutoresizingMaskIntoConstraints = false
-        bankSelectionLabel.topAnchor.constraint(equalTo: countrySelectionContainer.bottomAnchor, constant: 25).isActive = true
-        bankSelectionLabel.leftAnchor.constraint(equalTo: self.leftAnchor, constant: Kevin.shared.theme.insets.left).isActive = true
-        bankSelectionLabel.widthAnchor.constraint(equalTo: self.widthAnchor).isActive = true
-        bankSelectionLabel.heightAnchor.constraint(equalToConstant: 20).isActive = true
+        addSubview(tableView)
         
-        bankTableView.backgroundColor = Kevin.shared.theme.generalStyle.primaryBackgroundColor
-        bankTableView.rowHeight = 76
-        bankTableView.dataSource = self
-        bankTableView.estimatedRowHeight = 76
-        bankTableView.allowsSelection = false
-        bankTableView.separatorStyle = .none
-        addSubview(bankTableView)
-        
-        bankTableView.translatesAutoresizingMaskIntoConstraints = false
-        bankTableView.topAnchor.constraint(equalTo: bankSelectionLabel.bottomAnchor, constant: 16).isActive = true
-        bankTableView.widthAnchor.constraint(equalTo: self.widthAnchor).isActive = true
+        tableView.translatesAutoresizingMaskIntoConstraints = false
+        tableView.topAnchor.constraint(equalTo: topAnchor).isActive = true
+        tableView.widthAnchor.constraint(equalTo: widthAnchor).isActive = true
+        tableView.bottomAnchor.constraint(equalTo: bottomAnchor).isActive = true
     }
     
     private func initAgreementLabel() {        
```

Comment:
I can't see agreement label on the screen. Feels like it is either outside the screen or under the continue button